docs/config.md

```diff
@@ -477,18 +497,18 @@ spec:
 
 ### Target Discovery Options
 
-If you wish to use only Cryostat's [Discovery Plugin API](https://github.com/cryostatio/cryostat/blob/801779d5ddf7fa30f7b230f649220a852b06f27d/docs/DISCOVERY_PLUGINS.md), set the property `spec.targetDiscoveryOptions.builtInDiscoveryDisabled` to `true` to disable Cryostat's built-in discovery mechanisms.
+If you wish to use only Cryostat's [Discovery Plugin API](https://github.com/cryostatio/cryostat-legacy/blob/main/docs/DISCOVERY_PLUGINS.md), set the property `spec.targetDiscoveryOptions.disableBuiltInDiscovery` to `true` to disable Cryostat's built-in discovery mechanisms.
```

Comment:
I think we should have the new docs on 3.0 Cryostat to discovery the Discovery Plugin?

Reply:
We don't have a full document for that anymore.

https://github.com/cryostatio/cryostat/blob/604ebf3222a42eca959f2f7979c7d11f4581e69f/schema/openapi.yaml#L1233

would be the closest equivalent, but it should be a link to `main` rather than a particular commit. But then linking to `main` means the line number is not stable.

And the schema isn't very informative right now anyway - see https://github.com/cryostatio/cryostat/issues/378 .

Reply:
I think the better approach would be to simply link to the OpenAPI schema YAML with a description like "find the Discovery Plugin section", and then work on https://github.com/cryostatio/cryostat/issues/378 so that the OpenAPI schema actually contains some endpoint descriptions and can act as a replacement for the old HTTP_API.md and DISCOVERY_PLUGINS.md hand-written documents.

Reply:
Right, a link to OpenAPI schema sounds good to me!